security/nodeagent/values.yaml

```diff
@@ -1,16 +1,22 @@
 #
 # nodeagent configuration
 #
-nodeagent:
-  enabled: false
-  image: node-agent-k8s
-  env:
-    # name of authentication provider.
-    CA_PROVIDER: ""
-    # CA endpoint.
-    CA_ADDR: ""  
-    # names of authentication provider's plugins.
-    Plugins: ""
+sdsagent:
+
+  # This can be used to specify a different certificate provider
+  #
+  # This replaces the env variable in 1.2 alpha - env variables are not meant to be exposed to users for supported
+  # configs.
+  caProvider: "Citadel"
```

Comment:
for citadel today, we need to setup ```VALID_TOKEN: true``` in env (since usenormaljwt==true in this PR, citadel haven't use trusworthy jwt), this will nodeagent keep envoy sds connection (without disconnect and expected an updated token in cert rotation)

Reply:
Wait, citadel doesn't support 'trustworthy JWT' ? I didn't test it because k8s didn't seem to support it.

Reply:
Citadel supports trustworthy JWT.